deviceModeInit.js

```diff
@@ -138,10 +253,10 @@ var rudderTracking = (function () {
       }
     });
 
-
     trackPageEvent();
     trackNamedPageView();
 
+    // TODO: Correct this listner
```

Comment:
Do we need to make only listner change or any other change is also required as I am seeing one more TODO: Add Support for product Search